advanced/config.yaml

```diff
@@ -0,0 +1,87 @@
+components:
+  schemas:
+    Advanced.Config:
+      description: |-
+        An optional container for the configuration of advanced features.
+      type: object
+      properties:
+        latency:
+          $ref: '#/components/schemas/Advanced.Config.Latency'
+        event:
+          $ref: '#/components/schemas/Advanced.Config.Event'
+        convergence:
+          $ref: '#/components/schemas/Advanced.Config.Convergence'
+
+    Advanced.Config.Latency:
+      description: |-
+        The optional container for latency metric configuration.
+      type: object
+      properties:
+        enable:
+          description: |-
+            True to enable latency metrics using timestamps.
+            Enabling this option could possibly change flow packet payloads as 
```

Comment:
Enabling this option requires enabling flow metrics which may affect the resultant packet payload due to additional instrumentation data.